src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -0,0 +1,160 @@
+{
+    internal static class TrustedSignersCommand
+    {
+        internal static void Register(CommandLineApplication app,
+                      Func<ILogger> getLogger,
+                      Action<LogLevel> setLogLevel)
+        {
+            app.Command("trust", trustedSignersCmd =>
+            {
+                CommandArgument command = trustedSignersCmd.Argument(
+                    "<command>",
+                    Strings.TrustCommandActionDescription,
+                    multipleValues: true);
+
+                CommandOption algorithm = trustedSignersCmd.Option(
+                    "--algorithm",
+                    Strings.TrustCommandAlgorithm,
+                    CommandOptionType.SingleValue);
+
+                CommandOption allowUntrustedRootOption = trustedSignersCmd.Option(
+                    "--allow-untrusted-root",
+                    Strings.TrustCommandAllowUntrustedRoot,
+                    CommandOptionType.NoValue);
+
+                CommandOption owners = trustedSignersCmd.Option(
+                    "--owners",
+                    Strings.TrustCommandOwners,
+                    CommandOptionType.MultipleValue);
+
+                CommandOption verbosity = trustedSignersCmd.Option(
+                    "-v|--verbosity",
+                    Strings.Verbosity_Description,
+                    CommandOptionType.SingleValue);
+
+                CommandOption configFile = trustedSignersCmd.Option(
+                    "--configfile",
+                    Strings.Option_ConfigFile,
+                    CommandOptionType.SingleValue);
+
+                trustedSignersCmd.HelpOption(XPlatUtility.HelpOption);
+                trustedSignersCmd.Description = Strings.TrustCommandDescription;
+
+                trustedSignersCmd.OnExecute(async () =>
+                {
+                    TrustCommand action;
+
+                    if (!command.Values.Any() || string.IsNullOrEmpty(command.Values[0]))
+                    {
+                        action = TrustCommand.List;
+                    }
+                    else if (!Enum.TryParse(command.Values[0], ignoreCase: true, result: out action))
+                    {
+                        throw new CommandLineArgumentCombinationException(string.Format(CultureInfo.CurrentCulture, Strings.Error_UnknownAction, command.Values[0]));
+                    }
+
+                    string name = null;
+
+                    if (command.Values.Count > 1)
+                    {
+                        name = command.Values[1];
+                    }
+
+                    string packagePath = null;
+                    string sourceUrl = null;
+                    string fingerprint = null;
+                    if (command.Values.Count() > 2)
+                    {
+                        if (action == TrustCommand.Author || action == TrustCommand.Repository)
+                        {
+                            packagePath = command.Values[2];
```

Comment:
I recently created this [PR#3993](https://github.com/NuGet/NuGet.Client/pull/3993) for `nuget.exe`, as you can see based on other trust related commands doesn't accept multiple files we decided to accept only 1 file at the time. So I believe it applies here too. Please note that experience didn't work for `nuget.exe` even before my change, so there is no backward compat issue nor breaking experience.

But `dotnet nuget verify` case is different since it's just outputting result on cli and not mutating state of machine config, so it's ok.